src/style.css

```diff
@@ -1,3 +1,91 @@
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  --main-color: #00acdc;
+}
+/*In general we should use "!important" very carefully.
+But in case of hidden element it useful*/
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  background-color: #fff;
+  justify-content: flex-end;
+}
+
+.logo {
+  display: flex;
+  position: absolute;
+  left: 50px;
+  align-self: center;
```

Comment:
just give padding for header and thats it! no need to give all this styles for logo